integration/test/roster_test.js

```diff
@@ -0,0 +1,285 @@
+        });
+      });
+
+      describe('hook', async function () {
+        it('`muted` of local attendee is `true`', async function () {
+          await localPage.checkAttendeeState(localAttendeeName, 'muted', true);
+        });
+
+        it('`muted` of remote attendee is `true`', async function () {
+          await remotePage.checkAttendeeState(localAttendeeName, 'muted', true);
+        });
+      });
+    });
+
+    describe('unmute', async function () {
+      before(async function () {
+        await localPage.toggleMicrophone();
+      });
+
+      describe('component', async function () {
+        it('local attendee has unmuted microphone icon', async function () {
+          await localPage.checkMicrophoneIcon(localAttendeeName, false);
+        });
+
+        it('remote attendee has unmuted microphone icon', async function () {
+          await remotePage.checkMicrophoneIcon(localAttendeeName, false);
+        });
+      });
+
+      describe('hook', async function () {
+        it('`muted` of local attendee is `false`', async function () {
+          await localPage.checkAttendeeState(localAttendeeName, 'muted', false);
+        });
+
+        it('`muted` of remote attendee is `false`', async function () {
+          await remotePage.checkAttendeeState(localAttendeeName, 'muted', false);
+        });
+      });
+    });
+  });
+
+  describe('Video', async function () {
+    describe('enable', async function () {
+      before(async function () {
+        await localPage.toggleVideo();
+      });
+
+      describe('component', async function () {
+        it('local attendee has enabled video icon', async function () {
+          await localPage.checkVideoIcon(localAttendeeName, true);
+        });
+
+        it('remote attendee has enabled video icon', async function () {
+          await remotePage.checkVideoIcon(localAttendeeName, true);
+        });
+      });
+
+      describe('hook', async function () {
+        it('`videEnabled` of local attendee is `true`', async function () {
+          await localPage.checkAttendeeState(localAttendeeName, 'videoEnabled', true);
+        });
+
+        it('`videEnabled` of remote attendee is `true`', async function () {
+          await remotePage.checkAttendeeState(localAttendeeName, 'videoEnabled', true);
+        });
+      });
+    });
+
+    describe('disable', async function () {
+      before(async function () {
+        await localPage.toggleVideo();
+      });
+
+      describe('component', async function () {
+        it('local attendee has disabled video icon', async function () {
+          await localPage.checkVideoIcon(localAttendeeName, false);
+        });
+
+        it('remote attendee has disabled video icon', async function () {
+          await remotePage.checkVideoIcon(localAttendeeName, false);
+        });
+      });
+
+      describe('hook', async function () {
+        it('`videEnabled` of local attendee is `false`', async function () {
```

Comment:
nit: `videEnabled` -> `videoEnabled`. Here and in below `it` as well.

Reply:
Addressed, thanks for catching it.